extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -39,6 +39,8 @@ class CornerstoneViewportService implements IViewportService {
   renderingEngine: Types.IRenderingEngine | null;
   viewportsInfo: Map<number, ViewportInfo>;
   viewportGridResizeObserver: ResizeObserver | null;
+  // TODO - get the right type here.
```

Comment:
I've made HangingProtocolService typescript now, but haven't created an interface yet for it.